src/citrine/resources/ara_definition.py

```diff
@@ -41,6 +41,7 @@ class AraDefinition(Resource["AraDefinition"]):
 
     @staticmethod
     def _get_dups(lst: List) -> List:
+        # Hmmn, this looks like a potentially costly operation?!
         return [x for x in lst if lst.count(x) > 1]
```

Comment:
```suggestion
    def _get_dups(lst: List) -> List:
        seen = set()
        dups = [x for x in lst if x in seen or seen.add(x)]  # seen.add returns None
        return dups
```
But perhaps this is an off-topic optimizaton

Reply:
I agree that's a better approach. I'm going to leave it out of the PR for now just to keep focused.